vmdb/app/models/miq_report/search.rb

```diff
@@ -14,7 +14,7 @@ def get_limit_offset(page, per_page)
   end
 
   def get_sqltable(assoc)
-    r = self.db_class.reflect_on_association(assoc.to_sym)
+    r = db_class.reflection_with_virtual(assoc.to_sym)
```

Comment:
The previous line before @tenderlove changed it to `reflect_on_association` was `.reflections[assoc.to_sym]`.  Simple switch, so nothing surprising there.

I *think* the purpose of `get_sqltable` is to find the real reflection object for a real table.  It didn't do virtual reflections before, and probably shouldn't now...or did it?

The caller is get_col_info which has the guard `if apply_sortby_in_search` where `apply_sortby_in_search = !self.db_class.table_name.nil?`.  Do ActsAsArModel records have a table_name? If not, then how did an ActsAsArModel make its way to get_sqltable?